### ISSUE TITLE
Training with 8 GPUs

### ISSUE THREAD

**Issue by shwoo93** [bowenc0221/panoptic-deeplab#6]
Thanks for sharing the nice code :)

For fast training, I would like to use 8GPUs.
In this case, should I increase the learning rate by (x2) and reduce the iteration number by half (or just leave it as it is)?

Thanks,

**Comment by bowenc0221** [comment]
I would suggest using the same settings (learning rate and iteration unchanged) except batch size (which is what I will do).
Poly lr schedule is different from sgd, we don't know how to properly modify learning rate and iteration yet.

**Comment by shwoo93** [comment]
Thanks

**Comment by superxiaoying** [comment]
when 8 GPUS, what is about your suggest IMG_PER_BATCH? @bowenc0221

**Comment by bowenc0221** [comment]
@superxiaoying All models in the MODEL_ZOO are trained with 8 GPUs now, so you can use their corresponding config file with 8 GPUs.
